linera-execution/src/committee.rs

```diff
@@ -311,15 +313,16 @@ impl Committee {
     }
 
     #[cfg(with_testing)]
-    pub fn make_simple(keys: Vec<ValidatorPublicKey>) -> Self {
+    pub fn make_simple(keys: Vec<(ValidatorPublicKey, AccountPublicKey)>) -> Self {
         let map = keys
             .into_iter()
             .map(|k| {
```

Comment:
I'd find it much more readable to destructure here:

```suggestion
            .map(|(validator_pub_key, account_pub_key)| {
```

Reply:
Done in [Review part 1.](https://github.com/linera-io/linera-protocol/pull/3377/commits/edfb7bc5c4ef0158034e5d34cceffd76b9cba919).